index.js

```diff
@@ -114,20 +116,28 @@ exports.getKeyIndexes = function (commandName, args, options) {
       }
     }
     break;
-  case 'zunionstore':
-  case 'zinterstore':
-    keys.push(0);
-    keyStop = Number(args[1]) + 2;
-    for (i = 2; i < keyStop; i++) {
+  case 'migrate':
+    for (i = 3; i < args.length - 1; i++) {
```

Comment:
Definitely, I misread some part in the docs.
But `null` and `undefined` would be replaced with a string in node_redis. ioredis converts them to an empty string, right?